scripts/sct_dmri_moco.py

```diff
@@ -141,13 +147,19 @@ def main():
         param.path_out = arguments['-ofolder']
     if '-r' in arguments:
         param.remove_temp_files = int(arguments['-r'])
+    if '-interleaved' in arguments:
+        param.interleaved = int(arguments['-interleaved'])
     param.verbose = int(arguments.get('-v'))
 
     # Update log level
     sct.init_sct(log_level=param.verbose, update=True)
 
     # run moco
-    moco_wrapper(param)
+    if param.interleaved == 1:
+        # split input data to two datasets (even and odd slices), run moco in each sub-dataset and merge back the data
+        split_to_odd_and_even(param)
```

Comment:
i would call it `moco_wrapper_interleaved`, because `split_to_odd_and_even` is doing more than just splitting: it splits, applies moco, and then merges back.